packages/lisk-cryptography/src/hash.ts

```diff
@@ -15,25 +15,27 @@
 import crypto from 'crypto';
 import { hexToBuffer } from './buffer';
 
-const cryptoHashSha256 = data => {
-	const hash = crypto.createHash('sha256');
-	hash.update(data);
-	return hash.digest();
+const cryptoHashSha256 = (data: Buffer): Buffer => {
+	const dataHash = crypto.createHash('sha256');
+	dataHash.update(data);
+
+	return dataHash.digest();
 };
 
-const hash = (data, format) => {
+const hash = (data: Buffer | string, format?: string): Buffer => {
 	if (Buffer.isBuffer(data)) {
 		return cryptoHashSha256(data);
 	}
 
-	if (typeof data === 'string') {
-		if (!['utf8', 'hex'].includes(format)) {
+	if (typeof data === 'string' && typeof format === 'string') {
```

Comment:
shouldn't we check `null` or `undefined` here?

Reply:
I don't understand why.

Reply:
ah i think it's ok